data/json/construction.json

```diff
@@ -17,6 +17,15 @@
         "post_special" : "done_deconstruct"
     },{
         "type" : "construction",
+        "description" : "Take Down Simple Furniture",
+        "category" : "FURN",
+        "required_skills" : [ [ "fabrication", 0 ] ],
+        "time" : 10,
+        "pre_flags" : "EASY_DECONSTRUCT",
+        "pre_special" : "check_deconstruct",
+        "post_special" : "done_deconstruct"
```

Comment:
I think you could use the `pre_note` ([e.g. `t_palisade_pulley`](https://github.com/DangerNoodle/Cataclysm-DDA/blob/6deb2fb390bb7564621dc05adaae4487cb954fe4/data/json/construction.json#L1243)) added by https://github.com/CleverRaven/Cataclysm-DDA/pull/20519 to perhaps describe the differences between this action and the regular "Deconstruct Furniture" action.

If there's room, it might be nice from a UI perspective to additionally provide a list of the available furniture that fits the bill of being "easy." Otherwise, it would be a matter of trial and error by the player to discover those.